airflow/executors/executor_loader.py

```diff
@@ -184,6 +192,31 @@ def validate_database_executor_compatibility(cls, executor: type[BaseExecutor])
         if engine.dialect.name == "sqlite":
             raise AirflowConfigException(f"error: cannot use SQLite with the {executor.__name__}")
 
+    @classmethod
+    def import_all_executor_classes(cls, validate=True) -> list[type[BaseExecutor]]:
```

Comment:
```suggestion
    def import_all_executor_classes(cls, *, validate: bool = True) -> list[type[BaseExecutor]]:
```

kwarg only probably makes sense here?

Reply:
After some further discussion with @potiuk we're only going to load and inject CLI commands for the currently configured executor, so I actually no longer need this method. I'll just scrap it altogether for now.

Reply:
Actually, I'll include both celery and kube. I think it's safe to assume that dependencies for both should be available.